contracts/HashRegistrarSimplified.sol

```diff
@@ -238,7 +242,38 @@ contract Registrar {
         }
         return len;
     }
+    
+    /** 
+     * @dev Creates a scheduled release for each name
+     * 
+     * Each name will be assigned a random date in which its auction 
+     * can be started, from 0 to 6 months
+     * 
+     * @param _hash The hash to start an auction on
+     * @param _timestamp The timestamp to query about
+     */
+     
+    function isAllowed(bytes32 _hash, uint _timestamp) constant returns (bool allowed){
```

Comment:
lol 
True. I made this function first and then had a hard time reverting it into the getAllowedTime and didn't realize I didn't need the original one anymore.